internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_provider_test.go

```diff
@@ -36,35 +36,6 @@ func TestAccFederatedSettingsIdentityProviderDS_samlBasic(t *testing.T) {
 	})
 }
 
-func TestAccFederatedSettingsIdentityProviderDS_oidcBasic(t *testing.T) {
```

Comment:
are we just moving this? or why are we deleting it?

Reply:
looks like it was just moved below

Reply:
We have moved this to the resource test. Now we can programmatically create OIDC IdPs so in the resource test we create it and do what was being done in this test